test/integration/stateTransition/StateTransitionFacade.spec.js

```diff
@@ -136,6 +151,7 @@ describe('StateTransitionFacade', () => {
       dataProviderMock.fetchDataContract.resolves(dataContract);
       dataProviderMock.fetchIdentity.resolves({
         type: Identity.TYPES.USER,
+        getPublicKeyById: this.sinonSandbox.stub().returns(identityPublicKey),
```

Comment:
There will be no such method here, as it resolves to the raw json

Reply:
What do u mean?